spec/features/visitors/visitor_signs_in_with_google_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+describe 'as a visitor' do
+  it "should click sign in and be logged in with google oauth" do
+    stub_omniauth
+
+    visit root_path
+    expect(page).to have_link("Sign in with Google")
+
+    click_link "Sign in with Google"
+    expect(page).to have_content("Welcome")
+    # expect(page).to have_link("Logout")
```

Comment:
I think we decided to not worry about logout functionality at the moment? Let's just remove this commented-out line so it's clear.